src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -2022,7 +2030,6 @@ async def handle_report_incident_submission_event(
 
     result = await client.views_update(
         view_id=result["view"]["id"],
-        hash=body["view"]["hash"],
```

Comment:
This is the only way I've been able to resolve the hash_conflict errors. Removing it seems counter intuitive, would be good to check this.